marm-mcp-server/marm_mcp_server/endpoints/memory.py

```
@@ -208,7 +210,7 @@ async def marm_smart_recall(request: SmartRecallRequest, http_request: Request):
 
         return final_response
     except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)